src/algebra/module/submodule_lattice.lean

```diff
@@ -226,6 +226,11 @@ lemma mem_Sup_of_mem {S : set (submodule R M)} {s : submodule R M}
   (hs : s ∈ S) : ∀ {x : M}, x ∈ s → x ∈ Sup S :=
 show s ≤ Sup S, from le_Sup hs
 
+lemma eq_zero_of_coe_mem_of_is_compl (hpq : is_compl p q) {a : p} (ha : (a : M) ∈ q) :
```

Comment:
Isn't this true more generally for
```suggestion
lemma eq_zero_of_coe_mem_of_disjoint (hpq : disjoint p q) {a : p} (ha : (a : M) ∈ q) :
```
It follows trivially from `submodule.disjoint_def`, which is in another file. Either proving it next to that lemma, or moving that lemma here, would be fine by me.